cmd/scollector/collectors/fastly.go

```diff
@@ -67,6 +68,7 @@ const (
 	fastlyScheduledMaintDesc       = "The number of currently scheduled maintenances. Does not include maintenance that is current active"
 	fastlyActiveScheduledMaintDesc = "The number of currently scheduled maintenances currently in progress. Includes the 'in_progress' and 'verifying'"
 	fastlyActiveIncidentDesc       = "The number of currently active incidents. Includes the 'investingating', 'identified', and 'monitoring' states."
+	fastlyMaxDuration              = 3 * time.Minute
```

Comment:
In general I'm not clear on this change and the purpose virtue of using context here. I think maybe http already has the timeout set on the client with https://github.com/bosun-monitor/bosun/blob/master/cmd/scollector/main.go#L90

This perhaps something @captncraig probably knows better than me.